app/Http/Controllers/Table/RoutesTablesController.php

```diff
@@ -196,28 +194,33 @@ public function formatItem($route_entry)
                 $item['inetCidrRouteDest'] = $route_entry->inetCidrRouteDest;
             }
         }
-        $item['inetCidrRouteIfIndex'] = 'ifIndex ' . $item['inetCidrRouteIfIndex'];
+        $item['inetCidrRouteIfIndex'] = 'ifIndex ' . $route_entry->inetCidrRouteIfIndex;
         if ($port = $route_entry->port()->first()) {
             $item['inetCidrRouteIfIndex'] = Url::portLink($port, htmlspecialchars($port->getShortLabel()));
         }
         $device = Device::findByIp($route_entry->inetCidrRouteNextHop);
+        $item['inetCidrRouteNextHop'] = $route_entry->inetCidrRouteNextHop;
         if ($device) {
             if ($device->device_id == $route_entry->device_id || in_array($route_entry->inetCidrRouteNextHop, ['127.0.0.1', '::1'])) {
                 $item['inetCidrRouteNextHop'] = Url::deviceLink($device, 'localhost');
             } else {
-                $item['inetCidrRouteNextHop'] = $item['inetCidrRouteNextHop'] . '<br>(' . Url::deviceLink($device) . ')';
+                $item['inetCidrRouteNextHop'] = $route_entry->inetCidrRouteNextHop . '<br>(' . Url::deviceLink($device) . ')';
             }
         }
+        $item['inetCidrRouteProto'] = $route_entry->inetCidrRouteProto;
         if ($route_entry->inetCidrRouteProto && $route_entry::$translateProto[$route_entry->inetCidrRouteProto]) {
             $item['inetCidrRouteProto'] = $route_entry::$translateProto[$route_entry->inetCidrRouteProto];
         }
+        $item['inetCidrRouteType'] = $route_entry->inetCidrRouteType;
```

Comment:
Looking a bit better, instead of building the array ad-hoc, you can do this:
```php
return [
    'inetCidrRouteType' => $route_entry::$translateType[$route_entry->inetCidrRouteType] ?? $route_entry->inetCidrRouteType,
];
```

Reply:
I tried to make it that way but with the try catch for the exceptions, it would be a mess. I could prepare all the values for the array (one variable for each returned value), and return the dynamically built array, but it would not improve readability.

In term of performance, I really think it does not make a difference nowerdays. 

I think we have a more compact and readable way of filtering these variables now, compared to before, and for sure more efficient network-wise.

Reply:
Mostly, I value readability over other things.
Sometimes compactness and readability go together.

if statements are poison for readability ;)

Reply:
Yep, but I really don't see how to avoid the ones that are still around now.